quixo-react-native/screens/GameScreen.js

```diff
@@ -34,68 +72,69 @@ const isInMovables = movables => ({ x, y }) =>
 const isSelectedCube = selectedCube => ({ x, y }) =>
   selectedCube && selectedCube.x === x && selectedCube.y === y;
 
-const fetchGame = (id, dispatch) => () => {
-  const fetchGameCall = async () => {
-    const payload = await getNewGame();
-    dispatch({ type: UPDATE_GAME, payload });
-  };
-  if (!id) {
-    fetchGameCall();
-  }
-};
+const isMyTurn = (myTeam, currentPlayer) => myTeam === currentPlayer;
 
-const fetchMovables = (id, dispatch) => () => {
-  const fetchMovablesCall = async () => {
-    const payload = await getMovables(id);
-    dispatch({ type: FETCH_MOVABLES, payload });
-  };
-  if (id) {
-    fetchMovablesCall();
+const getInstructions = (team, isPlaying) => {
+  const isSpectator = team === NEUTRAL_VALUE;
+  if (isSpectator) {
+    return (
+      <View style={styles.instructions}>
+        <Text style={styles.instructionsText}>You're a spectator </Text>
```

Comment:
Apostrophe are breaking GitHub diff. Nothing to really worry about, but it's nice if you wrap "You're" with a template literal next time. `{"You're"}`